crime-evidence/src/main/java/uk/gov/justice/laa/crime/evidence/controller/CrimeEvidenceController.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.justice.laa.crime.evidence.controller;
+
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.justice.laa.crime.evidence.builder.CrimeEvidenceDTOBuilder;
+import uk.gov.justice.laa.crime.evidence.dto.CrimeEvidenceDTO;
+import uk.gov.justice.laa.crime.evidence.dto.ErrorDTO;
+import uk.gov.justice.laa.crime.evidence.model.common.ApiCalculateEvidenceFeeRequest;
+import uk.gov.justice.laa.crime.evidence.model.common.ApiCalculateEvidenceFeeResponse;
+import uk.gov.justice.laa.crime.evidence.service.EvidenceService;
+
+import javax.validation.Valid;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("api/internal/v1/evidence")
+@Tag(name = "Crown Evidence", description = "Rest API for Crown Evidence.")
```

Comment:
I think there's a type here... should this be **Crime** Evidence instead of **_Crown_** Evidence?